src/main/java/gift/filter/JwtFilter.java

```diff
@@ -0,0 +1,53 @@
+package gift.filter;
+
+import gift.util.UserUtility;
+import io.jsonwebtoken.Claims;
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+public class JwtFilter implements Filter {
+
+    private String tokenPrefix;
+    private UserUtility userUtility;
+
+    public JwtFilter(String tokenPrefix, UserUtility userUtility) {
+        this.tokenPrefix = tokenPrefix;
+        this.userUtility = userUtility;
+    }
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+        Filter.super.init(filterConfig);
```

Comment:
이미 상속받은 상태라서 
```suggestion
        super.init(filterConfig);
```
이렇게 업데이트 하는게 좋지 않을까요?

Reply:
해당 메서드가 디폴트 메서드라 Filter.super.init() 방식으로만 불러올 수 있지 않나요?? super.init() 방식으로는 오류가 발생합니다...

Reply:
아하 제가 미처 파악을 못했군요 확인 감사합니다 :)